src/pages/about.astro

```diff
@@ -45,9 +45,10 @@ const heroData = {
           <li>is publicly available to read;</li>
           <li>
             allows use, modification, and redistribution with minimal restrictions
-            to protect the producer’s business model; and
+            to protect the producer’s business model;
           </li>
-          <li>undergoes delayed Open Source publication (DOSP).</li>
+          <li>undergoes delayed Open Source publication (DOSP); and</li>
+          <li>does not put an undue burden on the user to monitor compliance.</li>
```

Comment:
I feel like this uses heavy language compared to the other points. It'll need clarification anyways (which we have below), so might as well use simpler wording imo. It also reads like an anti-feature compared to the others that are more like features, or at least positives.

Maybe something simple like "is easy to comply with"?